medmodels/treatment_effect_estimation/analysis_modules/estimate.py

```diff
@@ -0,0 +1,407 @@
+            num_treat_false,
+            num_control_true,
+            num_control_false,
+        ) = self._compute_subject_counts(medrecord=medrecord)
+
+        subject_counts = {
+            "treatment_true": num_treat_true,
+            "treatment_false": num_treat_false,
+            "control_true": num_control_true,
+            "control_false": num_control_false,
+        }
+
+        return subject_counts
+
+    def relative_risk(self, medrecord: MedRecord) -> float:
+        """
+        Calculates the relative risk (RR) of an event occurring in the treatment group
+        compared to the control group. RR is a key measure in epidemiological studies
+        for estimating the likelihood of an event in one group relative to another.
+
+        The interpretation of RR is as follows:
+        - RR = 1 indicates no difference in risk between the two groups.
+        - RR > 1 indicates a higher risk in the treatment group.
+        - RR < 1 indicates a lower risk in the treatment group.
+
+        Preconditions:
+            - Subject counts for each group must be non-zero to avoid division by zero
+                errors.
+
+        Args:
+            medrecord (MedRecord): The MedRecord object containing the data.
+
+        Returns:
+            float: The calculated relative risk between the treatment and control
+                groups.
+
+        Raises:
+            AssertionError: If the preconditions are not met, indicating a potential
+                issue with group formation or subject count retrieval.
+        """
+        (
+            num_treat_true,
+            num_treat_false,
+            num_control_true,
+            num_control_false,
+        ) = self._compute_subject_counts(medrecord=medrecord)
+
+        return (num_treat_true / (num_treat_true + num_treat_false)) / (
+            num_control_true / (num_control_true + num_control_false)
+        )
+
+    def odds_ratio(self, medrecord: MedRecord) -> float:
+        """
+        Calculates the odds ratio (OR) to quantify the association between exposure to a
+        treatment and the occurrence of an outcome. OR compares the odds of an event
+        occurring in the treatment group to the odds in the control group, providing
+        insight into the strength of the association between the treatment and the
+        outcome.
+
+        Interpretation of the odds ratio:
+        - OR = 1 indicates no difference in odds between the two groups.
+        - OR > 1 suggests the event is more likely in the treatment group.
+        - OR < 1 suggests the event is less likely in the treatment group.
+
+        Preconditions:
+            - Subject counts in each group must be non-zero to ensure valid
+                calculations.
+
+        Args:
+            medrecord (MedRecord): The MedRecord object containing the data.
+
+        Returns:
+            float: The calculated odds ratio between the treatment and control groups.
+
+
+        Raises:
```

Comment:
Should probably be the same Raises docstring as the other functions. Or at least not so ambiguous